modules/nlu/src/backend/api.ts

```diff
@@ -107,56 +74,74 @@ export default async (bp: typeof sdk, state: NLUState) => {
     }
   })
 
-  // TODO move this in intent router
   router.get('/intents', async (req, res) => {
-    res.send(await (state.nluByBot[req.params.botId].engine1 as ScopedEngine).storage.getIntents())
+    const { botId } = req.params
+    const ghost = bp.ghost.forBot(botId)
+    const intentDefs = await getIntents(ghost)
+    res.send(intentDefs)
```

Comment:
Maybe just a personal preference, but I find this a bit too much verbose, like if you only use botId once, it make more sense to keep it compact on the same line. 

Also, i'm not a fan of placeholder variables, unless its name brings something useful, it's just one more thing that need to be garbage collected.


```js
const ghost = bp.ghost.forBot(req.params.botId)
 res.send(await getIntents(ghost))
```

Reply:
Don't you thing that express stores the payload in a variable anyway ><  that's gotta be garbage collected anyways. I can do the changes I don't mind at all.

Reply:
Don't you thing that express stores the payload in a variable anyway ><  that's gotta be garbage collected anyways. I went kindof fast with some copy paste art that's probably why you see those "placehoder" variables. Also, those variables improves debugability, clarity and it avoids trainracks.

In any case, I can do the changes I don't mind at all. Thanks for your comment

Reply:
Good point for the debuggability. I should use it more often x)